libs/neo4j/tests/integration_tests/chains/test_graph_database.py

```diff
@@ -4,10 +4,10 @@
 from unittest.mock import MagicMock
```

Comment:
Detail: I think there is no `__init__.py` file in this directory, is this normal?

Reply:
Have added one